### PR TITLE
basic webxdc blog post with new resources/apps/spec

### PR DESCRIPTION
**Pull request by hpk42**
TODO: 
- [x] add screenshots/images for each section 
 
- [x] streamline story line some more

### CODE REVIEW

File: _posts/2024-02-17-webxdc-m3.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+a central authority to resolve conflicts and "tell the truth". 
+
+## Pixel: minimal CRDT-implementation for collaborative pixel-drawing (2.8 KB)
+
+Writing a webxdc app that offers a consistent user experience 
+can be done in few code and without pulling in dependencies. 
+The [Pixel](https://codeberg.org/webxdc/pixel) app allows 
+to set and clear pixels for anyone in a chat,
+and it resolves any concurrency conflicts by using "Lamport Clocks". 
+It does all of that in 150 lines of Javascript code without dependenciers,
```
Comment (ansuz) [inline _posts/2024-02-17-webxdc-m3.md]: typo: dependenciers -> dependencies

File: _posts/2024-02-17-webxdc-m3.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+proving that CRDTs are fundamentally less complex than you might think. 
+
+## A basic cross-platform Editor using Prosemirror and yjs (148KB)
+
+In around 400 lines of Code, 
+the [webxdc/editor](https://codeberg.org/webxdc/editor/src/branch/main/src) 
+implements an end-to-end encrypting offline-first editor
+that you can use in any chat group to jointly edit a document. 
+The editor uses the [y-webxdc provider](https://www.npmjs.com/package/y-webxdc) 
+which integrates webxdc's send/receive API with [yJS](https://yjs.dev/#features),
```
Comment (ansuz) [inline _posts/2024-02-17-webxdc-m3.md]: I believe they typically write "Yjs" rather than "yJS"

**Comment by ansuz** [comment]
looks good :+1:

File: _posts/2024-02-17-webxdc-m3.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+It's also fine -- there is a carpenter among us who freshly learned 
+HTML and Javascript and already has three apps in the store :)
+
+However, more experience and focus on user-facing web apps,
+including improving tooling, writing apps, helping newcomers etc. 
+would be very welcome. 
+If you are interested, or know someone who might be interested, 
+also in a paid contract, then please get in touch maybe 
+[by submitting an app](https://codeberg.org/webxdc/xdcget/src/branch/main/SUBMIT.md) 
+to indicate you are serious or playful -- ported casual games are also always welcome! :)
```
Comment (missytake) [inline _posts/2024-02-17-webxdc-m3.md]:
```suggestion
to indicate whether you are serious or playful -- ported casual games are also always welcome! :) 
```

File: _posts/2024-02-17-webxdc-m3.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+The refined [development docs page](https://webxdc.org/docs) 
+now begins with a "getting-started" page 
+that allows to create and run your first web app within minutes. 
+
+The docs generally got re-arranged and the 
+[webxdc specification](https://webxdc.org/docs/spec/index.html)
+now has table-of-content sections to ease discovery. 
+
+No worries, it's still a very small spec you can read in a quarter hour :)
```
Comment (missytake) [inline _posts/2024-02-17-webxdc-m3.md]:
```suggestion
No worries, it's still a very small spec you can read in quarter of an hour :) 
```

**Review by missytake — APPROVED** [review]
Love it, really breaks it down to the point. Some minor suggestions:

File: _posts/2024-02-17-webxdc-m3.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+and have [evolving experimental branches](https://github.com/deltachat/deltachat-core-rust/pull/5041) 
+to let webxdc apps use realtime P2P communication by means of [Iroh.network](https://iroh.network),
+one of the most interesting Rust efforts to come out of the ashes of Web3. 
+
+While Delta Chat developers co-evolve the webxdc spec and implementations,
+they typically focus on releasing Delta Chat native messenger apps on all platforms and stores.
+Developing webxdc apps themselves only happens on the "side". 
+This is not meant to say anything against doing a little app on the side!
+In fact, there is a carpenter by trade among us who freshly learned 
+HTML and Javascript and already has three apps in the store :)
```
Comment (missytake) [inline _posts/2024-02-17-webxdc-m3.md]:
```suggestion
HTML and Javascript and already has published three apps in the store :) 
```

File: _posts/2024-02-17-webxdc-m3.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+Writing a webxdc app that offers a consistent user experience 
+can be done in few code and without relying on any dependencies. 
+The [Pixel](https://codeberg.org/webxdc/pixel) app allows 
+to set and clear pixels for anyone in a chat,
+and it resolves any concurrency conflicts by using "Lamport Clocks". 
+It does that in 150 lines of plain Javascript code 
+proving that CRDTs are fundamentally less complex than you might think. 
+
+## A basic cross-platform Editor using Prosemirror and yJS (148KB)
```
Comment (missytake) [inline _posts/2024-02-17-webxdc-m3.md]:
```suggestion
## A basic cross-platform Editor using Prosemirror and Yjs (148KB)
```

**Review by missytake — APPROVED** [review]
two more suggestions ;)

**Comment by r10s** [comment]
very nice! i just read through again, did not had comments or suggestions :)

**Pull request merged**